api/src/movie/dao/movie.dao.ts

```diff
@@ -57,6 +57,17 @@ export class MoviesDao {
     });
   }
 
+  getWatchedListMovies(listId?: string, userId?: string) {
```

Comment:
can we call this `getMovies`?

Reply:
I just realized we already have `getMovies`. We should combine it with this one.